app/components/SendModal/index.js

```diff
@@ -192,13 +252,25 @@ class SendModal extends Component {
           <div className="send__to">
             <div className="send__label">{t('sendToLabel')}</div>
             <div className="send__input" key="send-input">
+              {hip2Input && <span className="send__prefix">{to ? (
+                <img src={LockSVG} />
+              ) : '@'}</span>}
               <input
                 type="text"
-                placeholder={t('recipientAddress')}
+                className={hip2Input ? (to ? 'send__input-hip2-success' : 'send__input-hip2') : null}
+                placeholder={hip2Input ? t('recipientHip2Address') : t('recipientAddress')}
```

Comment:
Do you think it's a good idea to indicate in the placeholder that HIP-2 resolution is enabled and possible?
Like instead of just `Recipient address` as default, it could say `Recipient address (start with @ for alias)` or similar.